### ISSUE TITLE
combing antigens_isotypes

### ISSUE THREAD

**Issue by kaiemjoy** [UCD-SEG/serocalculator#99]
Request from JS: When specifying multiple antigen_isotypes add an option to run separately and/or combined

**Comment by d-morrison** [comment]
I think this might already be possible, simply by adding `antigen_iso` as a stratifying variable (needs to be tested).

**Comment by kaiemjoy** [comment]
Yes, it works and will run the MLE for them combined but not individually and combined
